modules/smart-agent_systemd-timers/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,11 @@
+module: systemd-timers
+name: heartbeat
+
+transformation: false
+aggregation: true
+
+signals:
+  signal:
+    metric: "gauge.substate.failed"
+rules:
+  minor:
```

Comment:
this is not usual to use such a low severity. in general we use critical one because it is related to availability according to https://github.com/claranet/terraform-signalfx-detectors/wiki/Notifications-binding#critical. can you explain the choice ?

Reply:
IMHO crons/timers are usually used for batch jobs, which in most cases are not critical. I don't see a use case where a cron/timer create an availability issue platform-wide. But if it better fits with the default values used elsewhere, I still can set a critical level by default, and can send the notification on BH alerting channel if my use case.

Reply:
I confess I am not sure but perhaps you are right and there are less "critical" cronjob than "minor" ones.
it is up to you so ;)